nms/packages/magmalte/app/components/network/DataPlanConfig.js

```diff
@@ -100,7 +100,7 @@ function DataPlanConfig(props: Props) {
         <TableCell>
           {profile.max_dl_bit_rate === DATA_PLAN_UNLIMITED_RATES.max_dl_bit_rate
             ? 'Unlimited'
-            : profile.max_dl_bit_rate / BITRATE_MULTIPLIER + ' Mbps'}
+            : (profile.max_dl_bit_rate * ONE_GB_TO_MB) / BITRATE_MULTIPLIER + ' Mbps'}
```

Comment:
this does not look right to me. The unit of `max_dl_bit_rate` is still BPS and this will show in Mbps, right?

Reply:
we divide by the bps BITRATE_MULTIPLIER, which is a GB, but the message here prints the rate in Mbps, so i converted Gbps to Mbps.

Reply:
this is also premature, because NMS right now doesnt have a way to define or edit a data plan, but i want to keep NMS upto date with the changes in subscriberdb.

Reply:
This is quite error prone. max_dl_bit_rate unit is Bps and you want to show it in Mbps. It does not make sense to me to multiply and divide again for a simple unit conversion. I still do not get why the BITRATE_MULTIPLIER is modified.

Reply:
Just to clarify i didnt make the decision to show it in Mbps. The existing code in NMS was doing that already, so i am just conforming to the existing change(https://github.com/magma/magma/blob/29b4ad6f0af118154ef1799cc0500a2a4865d399/nms/packages/magmalte/app/components/network/DataPlanConfig.js#L104)

I can change the BITRATE_MULTIPLIER to its original value, if most data rates will be in Mbps, but since we dont have a UI for that, i am not sure, how it looks like? in swagger API i defined the data rate in Bps, then should we have dataRate in Bps in NMS UI too? According to the current code, it looks like we are expecting the NMS UI to take the data rate in Mbps.